src/main/resources/config.yml

```diff
@@ -35,6 +35,9 @@ player-names-uwuify: false
 # this settings is very dangerous
 books-uwuify: false
 
+# this setting is for the uwuify commands
+allow-toggle: true
```

Comment:
could we do a permission for allowing toggle?

Reply:
Will do 👍🏻